### ISSUE TITLE
Fails to launch

### ISSUE THREAD

**Issue by h313** [sidneys/pb-for-desktop#12]
Right after `npm install`. `appRootPath` seems to be an object not a string.

Traceback:
```
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received { resolve: [Function: resolve],
  require: [Function: require],
  toString: [Function: toString],
  setPath: [Function: setPath],
  path: '/usr/lib/node_modules/pb-for-desktop' }
    at assertPath (path.js:7:11)
    at Object.join (path.js:1211:7)
    at Object.<anonymous> (/usr/lib/node_modules/pb-for-desktop/bin/cli.js:29:34)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:420:7)
```

**Comment by snus-kin** [comment]
+1 getting this exact same error
edit:
Also same error on desktop

Both running arch

**Comment by sidneys** [comment]
Thanks for the heads up. The CLI module is currently not at all functional due to issues with app-root-path. High priority for the next release.

**Comment by sidneys** [comment]
Fixed since v2.9.976.
